tests/bluetooth/bsim/host/adv/encrypted/css_sample_data/src/main.c

```diff
@@ -0,0 +1,65 @@
+/* Copyright (c) 2023 Nordic Semiconductor ASA
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "bs_types.h"
+#include "bs_tracing.h"
+#include "bstests.h"
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(bt_bsim_ead_sample_data, CONFIG_BT_EAD_LOG_LEVEL);
```

Comment:
I'm wondering if this should use this same log level, as the library. 

Surely they are related, but it may be the case that we want to log debug information from the test, without logging debug information from the stack. 

For now it's not a problem, but just something I was wondering :)